RELEASE.md

```diff
@@ -14,7 +14,48 @@ Further details on building, testing and use can be found in [README.md](https:/
 
 ## Release notes
 
-This is version 0.7.1-dev of oqs-provider which continues from the earlier 0.7.0 release. This release is fully tested to be used in conjunction with the main branch of [liboqs](https://github.com/open-quantum-safe/liboqs) and is guaranteed to be in sync with v0.12.0 of `liboqs` as and when released.
+This is version 0.8.0-rc1 of oqs-provider which continues from the earlier 0.7.0 release. This release is fully tested to be used in conjunction with the main branch of [liboqs](https://github.com/open-quantum-safe/liboqs) and is guaranteed to be in sync with v0.12.0 of `liboqs`.
+
+### Deprecation notice
+
+This is to notify users of Kyber (Round 3 version) to switch to the ML-KEM (FIPS 203 final version) as support for Kyber will be removed with the next release of liboqs and oqsprovider.
+
+The addition of ML-DSA FIPS 204 final version to liboqs v0.12.0 has introduced a new signature API which includes a context string parameter. The liboqs team is planning to remove the old version of the API without a context string in the next release to streamline the API and bring it in line with NIST specifications. Users who have an opinion on this removal are invited to provide input at [liboqs #2001](https://github.com/open-quantum-safe/liboqs/issues/2001).
+
+### Security considerations
+
+* CVE-2024-54137: The associated liboqs v0.12.0 release fixed a bug in HQC decapsulation that leads to incorrect shared secret value during decapsulation when called with an invalid ciphertext. Thank you to Célian Glénaz and Dahmun Goudarzi from Quarkslab for identifying the issue.
+
+### What's New
+
+In addition to improving testing, CI, and fixing platform specific build issues this release of oqs-provider:
+
+* Updates IANA code points for ML-KEM.
+* Adds support for ML-DSA (FIPS 204 final version).
+* Adds support for context strings (when built against an OpenSSL version also supporting that feature).
+* Updates the implementation of draft-ietf-lamps-pq-composite-sigs from version 01 to version 02.
+* Adds a SBOM template in the CycloneDX 1.6 format.
```

Comment:
DTLS1.3 support also is noteworthy in my opinion, e.g., "Adds support for DTLS1.3". In this case, it's for a feature branch only but no code changes will be required as and when this gets enabled in OpenSSL, so not sure how to phrase that restriction (as outside our control).

Reply:
Perhaps "Adds support for DTLS 1.3 (pending support in OpenSSL)"?

Reply:
"pending support" sounds "too pessimistic"/as if this is not available -- which it kind of is: If you build the feature branch you have it (though admittedly not yet fully finished)... I also thought about "(if enabled)" but that's also not quite cutting it ("too optimistic").... Given (only?) those options, though, I'd go with yours @SWilson4 : It's more cautious and hence, more appropriate.

Reply:
"requires OpenSSL 3.5" is clearer and may age better? (assuming it will be part of that release)

Reply:
If there is consensus I shall proceed with "Adds support for DTLS 1.3 when built against OpenSSL versions >= 3.5. Of course, only if OpenSSL 3.5 is the release which will support this feature; I couldn't learn much with cursory search, could you please confirm this @baentsch? If the exact release is uncertain at the moment I shall proceed with Spencer's suggestion.

Reply:
I don't think this is guaranteed to be included in 3.5. What's your take @fwh-dc ? As I said above, I also support @SWilson4 's wording.

Reply:
I support @SWilson4's wording. I wouldn't mention Openssl version numbers before it is set in stone.